doc/modules/motion_correction.rst

```diff
@@ -134,11 +134,11 @@ Low-level API
 -------------
 
 All steps (**activity profile**, **motion inference**, **motion interpolation**) can be launched with distinct functions.
-This can be useful to find the good method and finely tune/optimize parameters at every steps.
+This can be useful to find the best method and finely tune/optimize parameters at each step.
```

Comment:
a good vs the best?

The plural is wrong but what about "The Police" song? Is there a a semantical distinction between each step and every step?

Reply:
I think best reads better :)

Reply:
@h-mayorquin You're right about the Police the song. This is the same distinction as todos los dias/cada dia vs tous les jours/chaque jour (I don't actually know the Italian equivalent). It's more stylistic than required. In English at least each step emphasizes the singular step in the process whereas every step doesn't emphasize that. I like the emphasis better, but it is not wrong to say every step. (step would be need to singular either way).

Reply:
Thanks for the explanation. You are completly right that in (my) Spanish I would say "Optimizar los parametros en cada paso" instead of 'Optimizar los parameteros en todos los pasos". Kudos.